[2024-05-01T23:32:02.011-0500] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-01T23:32:02.019-0500] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_bash_operator.runme_1 scheduled__2024-05-01T00:00:00+00:00 [queued]>
[2024-05-01T23:32:02.022-0500] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_bash_operator.runme_1 scheduled__2024-05-01T00:00:00+00:00 [queued]>
[2024-05-01T23:32:02.022-0500] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-01T23:32:02.027-0500] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): runme_1> on 2024-05-01 00:00:00+00:00
[2024-05-01T23:32:02.029-0500] {standard_task_runner.py:63} INFO - Started process 34812 to run task
[2024-05-01T23:32:02.032-0500] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_1', 'scheduled__2024-05-01T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', '/Users/nestoralfaro/Desktop/pbi-scenario-two/py_env/lib/python3.9/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/var/folders/14/31_1v10s5nvcqyswsggcn5vh0000gn/T/tmpiz35kjrf']
[2024-05-01T23:32:02.032-0500] {standard_task_runner.py:91} INFO - Job 3: Subtask runme_1
[2024-05-01T23:32:02.049-0500] {task_command.py:426} INFO - Running <TaskInstance: example_bash_operator.runme_1 scheduled__2024-05-01T00:00:00+00:00 [running]> on host nestors-macbook.attlocal.net
[2024-05-01T23:32:02.071-0500] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='example_bash_operator' AIRFLOW_CTX_TASK_ID='runme_1' AIRFLOW_CTX_EXECUTION_DATE='2024-05-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-01T00:00:00+00:00'
[2024-05-01T23:32:02.072-0500] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-01T23:32:02.072-0500] {subprocess.py:63} INFO - Tmp dir root location: /var/folders/14/31_1v10s5nvcqyswsggcn5vh0000gn/T
[2024-05-01T23:32:02.073-0500] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "example_bash_operator__runme_1__20240501" && sleep 1']
[2024-05-01T23:32:02.075-0500] {subprocess.py:86} INFO - Output:
[2024-05-01T23:32:02.077-0500] {subprocess.py:93} INFO - example_bash_operator__runme_1__20240501
[2024-05-01T23:32:03.086-0500] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-01T23:32:03.091-0500] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-01T23:32:03.129-0500] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_1, execution_date=20240501T000000, start_date=20240502T043202, end_date=20240502T043203
[2024-05-01T23:32:03.155-0500] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-01T23:32:03.174-0500] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-01T23:32:03.174-0500] {local_task_job_runner.py:222} INFO - ::endgroup::
